ui/app/styles/components/calendar-widget.scss

```diff
@@ -82,17 +58,3 @@ $dark-gray: #535f73;
   grid-gap: 0.7rem;
   padding: 7px 28px 28px 28px;
 }
-
-// for modal-dialog tooltips
-.calendar-tooltip {
-  background-color: $ui-gray-700;
-  color: $white;
-  font-size: $size-8;
-  padding: $spacing-10;
-  border-radius: $radius-large;
-  width: 141px;
-}
-
-.ember-modal-dialog {
```

Comment:
this component also used in the chart components - can you test that removing this still makes the tooltip appear okay there? I'm not sure if adding it in the first place was necessary

Reply:
The chart tooltips still show up fine!